pkg/provider/containers.go

```diff
@@ -108,6 +105,8 @@ func (c *Container) SetPreflightResults(preflightImageCache map[string]plibRunti
 	logger := stdr.New(checklogger)
 	ctx = logr.NewContext(ctx, logger)
 
+	check := plibContainer.NewCheck(c.Image, opts...)
```

Comment:
For reviewers, I made this change to make it look exactly like the `container.SetPreflightResults` func.